tests/cli/test_install.py

```diff
@@ -41,6 +44,15 @@ def assert_wrote_file_contents(mocker):
     )
 
 
+def assert_wrote_real_file_contents(test_dir, name, push_api_key):
```

Comment:
Do we want to keep both this and `assert_wrote_file_contents`? (the mocker-based version, above)

(OK with keeping it, just making sure it's intentional)

Reply:
The other one is still used by other specs. I'd be okay with rewriting them all, but not doing that in this PR.